### PR TITLE
Issue #223 fix.

### PR DESCRIPTION
**Pull request by ticketsvl**
Didn't test local, but must work anyway. 
AFAIK, only maintainers can create tags/releases, so task 6 - undone.

### CODE REVIEW

File: bot/skills/kozula.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+
+def kozula(update: Update, context: CallbackContext):
+    request = requests.get(CBR_URL)
+    root = ET.fromstring(request.content)
+    for child in root:
+        if child.attrib["ID"] == "R01235":
+            usd_rate = float(child.find("Value").text.replace(",", "."))
+
+    logger.info("USD rate: %s", usd_rate)
```
Comment (egregors) [inline bot/skills/kozula.py:28]: Looks like the variable 'usd_rate' might be referenced before the assignment. What if we do not get 'usd_rate' for some reason (network spike for example)? 

Maybe, we should care about this kind of case. 
Shall we?

**Review by egregors — CHANGES_REQUESTED** [review]
Wow! @ticketsvl looks great! Thanks a lot 🔥 🔥 🔥 🔥 🔥 
Could you please just check my little improvement suggestion in the diff's comments?

I have already fixed `black` error, so don't forget to pull the last version ^^

File: bot/skills/kozula.py (hunk 1 above)
Comment (ticketsvl) [reply bot/skills/kozula.py]: Yes, we should.
I`ve merged online by misclick, but It seems ok?

Comment (ticketsvl) [reply bot/skills/kozula.py]: Oh, I though you fixed `black` error about quotes in it's settings? But It still decline. It's too important to do it manually and always in the same way? Position of import module must be in right place too? :) Looks funny.

**Comment by ticketsvl** [comment]
Looks good. You need anything more to merge?
I see "1 change requested" but I've already pushed "resolve conversation".

File: bot/skills/kozula.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import xml.etree.ElementTree as ET
+import requests
+from mode import cleanup_queue_update
+
+from telegram import Update
+from telegram.ext import Updater, CommandHandler, CallbackContext
+
+logger = logging.getLogger(__name__)
+
+KOZULA_RATE = 600000
```
Comment (lamak) [inline bot/skills/kozula.py]: Should we place variable in env?

Comment (egregors) [reply bot/skills/kozula.py]: Heh, kozula updated so infrequently. Not such a big deal.

File: bot/skills/kozula.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+def add_kozula(upd: Updater, handlers_group: int):
+    logger.info("registering tree handlers")
+    dp = upd.dispatcher
+    dp.add_handler(CommandHandler("kozula", kozula, run_async=True), handlers_group)
+
+
+def kozula(update: Update, context: CallbackContext):
+    try:
+        request = requests.get(CBR_URL)
```
Comment (egregors) [inline bot/skills/kozula.py:23]: To be honest, looks like part with an HTTP request could be extracted. And it should call through some cache.

File: bot/skills/kozula.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+                text = (
+                    f"Текущий курс Козули в RUB: {kozula_rub_formated}\n"
+                    f"Текущий курс Козули в USD ~ {kozula_usd_formated}\n"
+                    f"Курс USD/RUB: {round(usd_rate, 2)}"
+                )
+            else:
+                logger.info("Error! Can't get USD rate!")
+                text = (
+                    f"Текущий курс Козули в RUB: {kozula_rub_formated}\n"
```
Comment (egregors) [inline bot/skills/kozula.py:42]: `kozula_rub_formated` still could be undefined if searching `R01235` will fail

**Review by egregors — APPROVED** [review]
Good job! I just left a few stylistic remarks. So, I gonna fix it right now by myself and merge the full feature.

**Pull request merged**